testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js

```diff
@@ -1066,6 +1068,46 @@ QUnit.module("api", moduleConfig, () => {
         assert.equal(keyDownSpy.callCount, 2, "keyDown event handled twice");
         assert.equal(keyUpSpy.callCount, 2, "keyUp event handled twice");
     });
+
+    QUnit.test("user onKeyPress event subscriptions fires a deprecation warning", function(assert) {
+        this.instance.dispose();
```

Comment:
😞

Reply:
@alexander-kotov-dx  use can extract this test to a separate module to avoid of unnecessary `beforeEach` callback